### PR TITLE
Add js test driver fix for Registry not being cleared after tests

### PR DESCRIPTION
**Pull request by germtb**

### CODE REVIEW

**Comment by germtb** [comment]
Fixes https://github.com/BladeRunnerJS/brjs/issues/1657

File: brjs-sdk/sdk/libs/javascript/br-locale/src/switcher.js [hunk 1]
```diff
@@ -32,6 +32,19 @@ module.exports = {
 		var normalizedPath = path === "/" ? "" : path.replace(/^\/?(.*?)(\/|\/index\.html)?$/, '/$1');
 		var localizedPath = normalizedPath + '/' + locale + ((fullyQualifiedPath) ? '.html' : '');
 
+		if (navigator.appName == 'Microsoft Internet Explorer') {
```
Comment (trueadm) [inline brjs-sdk/sdk/libs/javascript/br-locale/src/switcher.js]: should be `===`

File: brjs-sdk/sdk/libs/javascript/br-locale/src/switcher.js [hunk 2]
```diff
@@ -32,6 +32,19 @@ module.exports = {
...
+			if (pageUrl.indexOf(":80") > -1) {
+				localizedPath = localizedPath.substring(1, localizedPath.length);
+			}
+			else if (host.indexOf(":80") > -1){
```
Comment (trueadm) [inline brjs-sdk/sdk/libs/javascript/br-locale/src/switcher.js]: shouldn't the `else if` be on the line above rather than a new line?

Comment (germtb) [inline brjs-sdk/sdk/libs/javascript/br-locale/src/switcher.js]: Not really, it's for the case where the navigator is not IE, but the host port is explicitly asked for.

**Pull request merged**